go.mod

```diff
@@ -183,7 +183,7 @@ require (
 	github.com/elastic/ebpfevents v0.6.0
 	github.com/elastic/elastic-agent-autodiscover v0.9.0
 	github.com/elastic/elastic-agent-libs v0.17.4
-	github.com/elastic/elastic-agent-system-metrics v0.11.4
+	github.com/elastic/elastic-agent-system-metrics v0.11.5-0.20241210092419-b0ba11859dc9
```

Comment:
This is temporary. Once I verify the fix one more time, I will release a patch and update go.mod